Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Use official Ubuntu 22.04 image as base
+FROM ubuntu:22.04
+
+# Update the repository for package indexes
+RUN apt-get update
+
+# Install dependeces
+RUN apt-get install -y \
+    python3 \
+    git \
+    python3-pip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Set work directoryI
+WORKDIR /work
+
+# Clone a Git repository using the provided credentials
+RUN git clone https://github.com/FabioRagosta/nmma/
```

Comment:
Should this line instead clone `https://github.com/nuclear-multimessenger-astronomy/nmma/`?

Reply:
yes it should, I forgot to change it before uploading. I used the forked repository for the trial.